game.py

```diff
@@ -43,13 +51,22 @@ def check_events(self):
                     self.DOWN_KEY = True
                 if event.key == pygame.K_UP:
                     self.UP_KEY = True
-
+                if event.key == pygame.K_LEFT:
+                    self.LEFT_KEY = True
+                if event.key == pygame.K_RIGHT:
+                    self.RIGHT_KEY = True
+                    
     def reset_keys(self):
-        self.UP_KEY, self.DOWN_KEY, self.START_KEY, self.BACK_KEY = False, False, False, False
+        self.LEFT_KEY, self.RIGHT_KEY,self.UP_KEY, self.DOWN_KEY, self.START_KEY, self.BACK_KEY = False, False, False, False, False, False
```

Comment:
The line is becoming quite long we might want to consider splitting it at least in two 
1. Right and Left
2. Up and Down

Reply:
OK, I'm splitting into 3 lines